src/proposals/proposalTypes/HybridProposal.sol

```diff
@@ -567,6 +737,14 @@ abstract contract HybridProposal is
             "Proposal state not executed"
         );
 
+        for (uint256 i = 0; i < moonbeamActions.length; i++) {
```

Comment:
I believe this check can be used for basic actions as well, so we can loop all actions here, and also, I think this should be done before executing